src/select/vec_searchable.rs

```diff
@@ -134,26 +138,62 @@ where
-                                top_width={*top_width}
-                                max_width={*max_width}
-                                max_height={*max_height}
-                                list={list}
-                                selected={Rc::clone(selected)}
-                                allow_empty={*allow_empty}
-                                default_all={false}
-                                sized_value={sized_value}
-                                parent_message={Message::Select(index)}
-                            />
+                            if cfg!(feature = "pumpkin") {
+                                <div class={field_class}>
+                                    <SelectSearchable<Self>
+                                        txt={ctx.props().txt.clone()}
+                                        language={ctx.props().language}
+                                        id={format!("{}-{index}-{}", ctx.props().id.clone(), self.rerender_serial)}
+                                        kind={kind}
+                                        title={title.clone()}
+                                        empty_msg={empty_msg.clone()}
+                                        top_width={*top_width}
+                                        max_width={*max_width}
+                                        max_height={*max_height}
+                                        list={list}
+                                        selected={Rc::clone(selected)}
+                                        allow_empty={*allow_empty}
+                                        default_all={false}
+                                        sized_value={sized_value}
+                                        parent_message={Message::Select(index)}
+                                    />
+                                </div>
+                            } else {
+                                <SelectSearchable<Self>
+                                    txt={ctx.props().txt.clone()}
+                                    language={ctx.props().language}
+                                    id={format!("{}-{index}-{}", ctx.props().id.clone(), self.rerender_serial)}
+                                    kind={kind}
+                                    title={title.clone()}
+                                    empty_msg={empty_msg.clone()}
+                                    top_width={*top_width}
+                                    max_width={*max_width}
+                                    max_height={*max_height}
+                                    list={list}
+                                    selected={Rc::clone(selected)}
+                                    allow_empty={*allow_empty}
+                                    default_all={false}
+                                    sized_value={sized_value}
+                                    parent_message={Message::Select(index)}
+                                />
```

Comment:
This part is duplicated, so I think it's better to declare it once and reuse it.